workflow/rules/diffexp.smk

```diff
@@ -262,6 +242,42 @@ rule logcount_matrix:
         "../scripts/sleuth-to-matrix.R"
 
 
+rule get_heatmap:
```

Comment:
Rename rule into plot_diffexp_heatmap and also rename the script accordingly.

Reply:
Done